web/modules/ftml-wasm/tests/ftml.ts

```diff
@@ -86,8 +84,12 @@ describe("ftml-wasm", () => {
     })
   })
 
-  it("renderHTML", async () => {
-    await lib.loading
+  // following two tests are failing, reason appears to be
+  // due to getrandom failing. I think this is an issue
+  // with how getrandom calls up to the browser for randomness.
+  // other tests don't fail, but these do, for some reason.
+
+  it.skip("renderHTML", () => {
```

Comment:
Test mode doesn't compile ftml in test mode right? If it did this issue wouldn't be present, but if you use release or debug you'll have this issue. I should add an option for parse settings to make random ID generation deterministic for testing.

Reply:
That would work, or calling `getrandom()` yourself to provide the seed, as that returns a `Result`, unlike the `from_entropy()` method which throws. I'm not sure what you should use as your seed if `getrandom()` fails, but considering our use case it really wouldn't matter that much what you use as the fallback.